main.py

```diff
@@ -1,10 +1,39 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players = json.load(file)
+
+        for nickname, data in players.items():
+            race, created = Race.objects.get_or_create(
+                name=data["race"]["name"],
+                defaults={"description": data["race"]["description"]}
+            )
+            for skill_data in data["race"]["skills"]:
+                Skill.objects.get_or_create(
+                    name=skill_data["name"],
+                    defaults={"bonus": skill_data["bonus"], "race": race}
+                )
+
+            guild = None
+            if data.get("guild"):
+                guild, created = Guild.objects.get_or_create(
+                    name=data["guild"]["name"],
+                    defaults={"description": data["guild"].get("description")}
+                )
+
+            Player.objects.create(
+                nickname=nickname,
+                email=data["email"],
+                bio=data["bio"],
+                race=race,
+                guild=guild
+            )
```

Comment:
You should close the file after reading an information.
```suggestion
    with open("players.json", "r") as file:
    players = json.load(file)

for nickname, data in players.items():
    race, created = Race.objects.get_or_create(
        name=data["race"]["name"],
        defaults={"description": data["race"]["description"]}
    )
    for skill_data in data["race"]["skills"]:
        Skill.objects.get_or_create(
            name=skill_data["name"],
            defaults={"bonus": skill_data["bonus"], "race": race}
        )

    guild = None
    if data.get("guild"):
        guild, created = Guild.objects.get_or_create(
            name=data["guild"]["name"],
            defaults={"description": data["guild"].get("description")}
        )

    Player.objects.create(
        nickname=nickname,
        email=data["email"],
        bio=data["bio"],
        race=race,
        guild=guild
    )
```